spec/integration/rate_limiting_spec.rb

```diff
@@ -7,10 +7,14 @@
   before do
     RateLimiter.enable
     RateLimiter.clear_all!
+
+    @old_value = Rails.application.config.consider_all_requests_local
+    Rails.application.config.consider_all_requests_local = false
```

Comment:
Why do we need to disable this? The specs are still passing for me after removing this line. :thinking: